IdentityCore/src/IdentityCore_Internal.h

```diff
@@ -47,4 +47,4 @@
 - (void)SETTER:(NSString *)value { [DICT setValue:[value copy] forKey:KEY]; }
 
 #define STRING_CASE(_CASE) case _CASE: return @#_CASE
-#define MSID_ENABLE_SSO_EXTENSION ((__IPHONE_OS_VERSION_MAX_ALLOWED >= 130000 || __MAC_OS_X_VERSION_MAX_ALLOWED >= 101500) && !MSID_EXCLUDE_WEBKIT)
```

Comment:
Minimum version for iOS is now 14.0 (140000) and for macOS is 10.15 (101500) so all checks like this have been removed as they are now always true